src/test/resources/Features/Login.Feature

```diff
@@ -0,0 +1,8 @@
+Feature: Login Feature
+
+  Scenario: Login Success
+    Given I open Login page
+    When I enter email "dagebeyehu@gmail.com"
+    And I enter password "yozca5uQiLxb!"
+    And I submit
+    Then I am logged in
```

Comment:
Please, rename file `Login.feature`

For example

![image](https://github.com/Test-Pro-Bootcamp/A49-QA-Automation-Framework/assets/104761482/42d29946-7536-41e9-9746-0f8f5574034b)